### ISSUE TITLE
sshfs 

### ISSUE THREAD

**Issue by ZoranBlue** [billziss-gh/sshfs-win#163]
The OS of my computer is Windows 10.

I managed to mount the folders of my Mac on my computer. I struggle to do the same with my WD My Cloud Mirror NAS.

With PuTTY everything works fine:
host: 192.168.1.70
port: 22
When the credential message prompts, I indicate the username and pass that I chose for the ssh connection of my NAS.
Everything works fine, I'm inside of the server.

With sshsf, I insert the exact same infos using but it doesn't works: 
net use T: \\sshfs\root@192.168.1.70

Then I enter the same credentials and I get after a couple of seconds: 
"System error 5 has occurred"
"Access denied"

Any idea about how to fix it?

**Comment by billziss-gh** [comment]
Does your NAS drive allow login as root? You should use the same username and password you used when using Putty.

**Comment by ZoranBlue** [comment]
Yes, with PuTTY, I can also login by indicating root@192.168.1.70 and then indicating my password.

**Comment by billziss-gh** [comment]
Does it work if you try to "Map Network Drive" from the Explorer instead of using `net use`?

The two methods are normally equivalent, but there are some differences in how they collect the authentication information.

**Comment by ZoranBlue** [comment]
I tried it in different ways:
\\sshfs\root@192.168.1.70
\\sshfs\sshd@192.168.1.70

Then I indicate my password but I can't log in.
In Putty I can log in when I indicate root and sshd.

Actually, I tried to map the drives of my Mac. It worked with net use but not with \\sshfs.

**Comment by billziss-gh** [comment]
Unfortunately I do not have a good answer for your problem. Both the Explorer "map network drive" and the `net use` command use the same "network provider" internally so it is perplexing that one works when the other does not. I am assuming here that you are executing both as a regular user and not an Administrator and that you have not changed any of the default sshfs settings.
